src/vue/modules/assets/my-assets-explorer/index.vue

```diff
@@ -1,131 +1,62 @@
 <template>
   <div class="my-assets-explorer">
     <template v-if="isLoaded">
-      <drawer :is-shown.sync="isDrawerShown">
-        <template v-if="isAssetFormShown">
-          <template slot="heading">
-            {{ 'assets.update-drawer-title' | globalize }}
-          </template>
-
-          <asset-form
-            :former="former"
-            @submitted="onAssetUpdate"
-          />
+      <template v-if="isLoadFailed">
```

Comment:
looking on `load()` method, seems that it's impossible to have both `isLoaded` and `isLoadFailed` set to true